packages/starlight/translations/cs.json

```
@@ -25,5 +25,18 @@
   "aside.caution": "Upozornění",
   "aside.danger": "Nebezpečí",
   "fileTree.directory": "Adresář",
-  "builtWithStarlight.label": "Postavené s Starlight"
+  "builtWithStarlight.label": "Postavené s Starlight",
+  "expressiveCode.copyButtonCopied": "Zkopírováno!",
+  "expressiveCode.copyButtonTooltip": "Kopíruj do schránky",
+  "expressiveCode.terminalWindowFallbackTitle": "Terminál",
+  "pagefind.clear_search": "Vyčistit",
+  "pagefind.load_more": "Načíst další výsledky",
+  "pagefind.search_label": "Vyhledat stránku",
+  "pagefind.filters_label": "Filtry",
+  "pagefind.zero_results": "Žádný výsledek pro: [SEARCH_TERM]",
+  "pagefind.many_results": "počet výsledků: [COUNT] pro: [SEARCH_TERM]",
```

Should this one be something like this?
```suggestion
  "pagefind.many_results": "[COUNT] výsledků pro: [SEARCH_TERM]",
```

---

Chatted [on Discord](https://discord.com/channels/830184174198718474/830184175176122389/1253098475696947281) and sounds like it’s intentional to get around a lack of support for proper plurals support in Pagefind’s interpolation 👍